pronto_mia/lib/ui/views/deployment_plan/overview/deployment_plan_overview_view.dart

```diff
@@ -22,71 +22,93 @@ class DeploymentPlanOverviewView extends StatelessWidget {
       ),
       builder: (context, model, child) => Scaffold(
         appBar: AppBar(
-          title: const Text('Einsatzpläne'),
-          actions: <Widget>[
-            IconButton(
-              icon: const Icon(Icons.admin_panel_settings),
-              tooltip: 'Administrator-Modus',
-              onPressed: () {
-                model.toggleAdminMode();
-                toggleAdminModeCallback();
-              },
-              color: (() {
-                if (model.adminModeEnabled) {
-                  return Colors.yellow;
-                } else {
-                  return null;
-                }
-              })(),
-            ),
-            IconButton(
-              icon: const Icon(Icons.account_circle),
-              tooltip: 'Benutzerprofil',
-              onPressed: () {},
-            ),
-          ],
+          automaticallyImplyLeading: false,
+          title: const Text(
+            'Einsatzpläne',

```

Comment:
Is the same title for both edit and normal view.

Reply:
Fixed.